smoketest/scripts/cli/test_service_monitoring_prometheus.py

```diff
@@ -55,13 +56,19 @@ def tearDown(self):
 
     def test_01_node_exporter(self):
         self.cli_set(base_path + ['node-exporter', 'listen-address', listen_ip])
+        self.cli_set(base_path + ['node-exporter', 'collectors', 'textfile'])
 
         # commit changes
         self.cli_commit()
 
         file_content = read_file(node_exporter_service_file)
         self.assertIn(f'{listen_ip}:9100', file_content)
 
+        self.assertTrue(os.path.isdir('/run/node_exporter/collector'))
```

Comment:
Do you think it may be worthwile to make this path a reusable constant?

Reply:
I've defined a constant in `service_monitoring_prometheus.py`, if it would be reusable from there then maybe, otherwise I don't think it is worthwhile to make it reusable, as I don't see much reason for changing it.